### ISSUE TITLE
Doesn't work anymore with new import * as style from './component.scss

### ISSUE THREAD

**Issue by Yegorich555** [atfzl/eslint-plugin-css-modules#98]
Webpack css-loader introduced breaking change from version 7.0: https://github.com/webpack-contrib/css-loader/releases/tag/v7.0.0
**Before:**
```js
import style from "./style.css";

console.log(style.myClass);
```

**After:**
```js
import * as style from "./style.css";

console.log(style.myClass);
```

**Expected**: eslint-plugin-css-modules throws error if style.myClass isn't defined when we use new import style `import * as style from "./style.css";`

**Actual**:  eslint-plugin-css-modules ignores such types of imports
